src/modules/Builder/components/PackageList/PackageList.jsx

```diff
@@ -46,6 +47,11 @@ export default class PackageList extends Component {
             });
     }
 
+    executeState(builderCallback) {
+        this.builderCallback = builderCallback.bind(this);
```

Comment:
It's enought to call the function passed to executeState method, without defining is as an attribute of the class instance.